app/views/layouts/application.html.erb

```diff
@@ -67,6 +67,23 @@
           "This is a new service – your #{ghwt_contact_mailto(label: "feedback", subject: "Feedback")} will help us to improve it.".html_safe
         end
       end %>
+
+      <% if impersonated_user %>
+        <%= render GovukComponent::NotificationBanner.new(title: 'Information', disable_auto_focus: true, classes: 'govuk-!-margin-top-2 govuk-!-margin-bottom-1') do |notification_banner|
+              notification_banner.slot(
+                :heading,
+                text: 'You are currently impersonating',
+                link_text: impersonated_user.email_address,
+                link_target: support_user_path(impersonated_user)
+              )
```

Comment:
Is it possible to do:

```You are impersonating [Full Name](/support/users/USER_ID) (email@email.com)
[Stop impersonating]```

?

Reply:
Oh yea sorry, totally forgot, will do now

Reply:
I've updated, please see screenshot